### DISCUSSION TITLE
Is it possible to calibrate DS3 Led indicator with new bought battery ?

### DISCUSSION THREAD

**Discussion by fraf** [nefarius/DsHidMini#176] [Q&A] [answered]
Hi everyone,

I bought an old sixaxis  (official) ps3 pad. And this one probably was sold in 2006 to 2008. Of course, fourteen years later, the battery are used and last no more than 5 hours. New battery on this kind of pad (without vibration) can last up to 30 hours ! The problem is that at mid used time lengh (let's say 15 hours), the led indicator is already blinking for next 15 hours. Hoo ! Seems that calibration is missing. I don't know if it is possible to calibrate the led indicators with my new battery , i.e. blinking when it really 1 hour left.

Thanks for answers.

**Comment by nefarius** [comment, marked as answer]
Hi!

Unfortunately you can not calibrate that, the controller itself reports some pre-defined states it thinks of the battery level, not a percentage. So if the controller deducts (from voltage drop) that it is low and close to dying, it will report that, even if that state lasts you for another day. That's the deal with legacy hardware, it wasn't intended for such a long life span but we still love it, right? 😜 

Cheers!

**Reply by fraf** [reply]
Ok. It is a pity that leds can't be software-driven. Thanks.
